### DISCUSSION TITLE
[FEATURE REQ] Notification priority for supported notificators

### DISCUSSION THREAD

**Discussion by ghost** [autobrr/autobrr#1674] [Ideas]
Would like to see the following if at all possible, thanks for considering, loving the software!

- ~~Ability to set notifications as filter action, would ideally allow for ability to control notification level on a per filter basis.~~ Covered by #602 
- Have notifications for important actions such as disconnections / reconnections / updates be sent at a higher priority level.

**Comment by zze0s** [comment]
Hi! Happy you like it!

> Ability to set notifications as filter action, would ideally allow for ability to control notification level on a per filter basis.

We have this already which is probably the same thing? https://github.com/autobrr/autobrr/issues/602

> Have notifications for important actions such as disconnections / reconnections / updates be sent at a higher priority level.

Could you extend on this a bit? Is there any specific Notificator using priorities? We just send it all, some of them different events.

**Comment by ghost** [comment]
> Hi! Happy you like it!
> 
> > Ability to set notifications as filter action, would ideally allow for ability to control notification level on a per filter basis.
> 
> We have this already which is probably the same thing? #602
> 
> > Have notifications for important actions such as disconnections / reconnections / updates be sent at a higher priority level.
> 
> Could you extend on this a bit? Is there any specific Notificator using priorities? We just send it all, some of them different events.

Yep, re #602 I must have missed that, definitely the same thing.

Well to me it seems like autobrr itself should send these sort of events (IRC connections/update) at a higher priority level (which you can control if using pushover for example). So I feel like all of those alerts should use a priority of 1 (or have the ability to choose your priority). If #602 was then implemented, I'd then like to be able to control the priority level of those on a per filter basis.

System Autobrr IRC disconnect/reconnect/update = Priority 1
Filter priority = 0

**Comment by zze0s** [comment]
Okay so it relates to Pushover then since Discord for example does not have the concept of priority? :+1:

**Comment by ghost** [comment]
> Okay so it relates to Pushover then since Discord for example does not have the concept of priority? 👍

Yes as it seems like the most versatile option at the moment and I have no intention of using discord for this purpose.
